incubator/quarkus/templates/kafka/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+
+services:
+
+  zookeeper:
+    image: strimzi/kafka:0.15.0-kafka-2.3.1
```

Comment:
Was here a reason why you upgraded this version? It seems to work, but it's a change from what's in the original quickstart. See the version targeting Quarkus 1.3.0.Final: https://github.com/quarkusio/quarkus-quickstarts/blob/296582a9ccc2393aa5ec79b2f3e56d87fc7d021d/kafka-quickstart/docker-compose.yaml and the current master version: https://github.com/quarkusio/quarkus-quickstarts/blob/master/kafka-quickstart/docker-compose.yaml. Both reference an older Kafka version.

Reply:
I used the `docker.compose-yaml` that we've used in the Spring and Node.js `kafka` templates. I didn't realise the one in the quickstart was backlevel.  Kafka 2.5 is out now so even this is out of date.

To be honest I don't think it hugely matters - this `docker-compose.yaml` is only going to be used for people who are experimenting with the template. Any real app will likely use a pre-existing Kafka that the developer already has access to.

Reply:
That's true, and it does seem to work for the experimenting.